about.html

```diff
@@ -0,0 +1,169 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&family=Poppins:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/about.css" />
+  </head>
+  <body>
+    <header>
+      <div class="mobile-menu-icon" id="mobile-menu">
+        <a href="#"
+          ><img src="images/mobile_menu.png" alt="Hamburger Icon"
+        /></a>
+      </div>
+      <nav class="social-link-top">
+        <ul>
+          <li>
+            <a href="#"
+              ><img src="images/facebook.png" alt="facebook icon"
+            /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/twitter.png" alt="twitter icon" /></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </nav>
+      <nav class="navbar-menu">
+        <div class="logo">
+          <a href="index.html"
+            ><img src="images/eta_logo.png" alt="ETA institution logo here"
+          /></a>
+        </div>
+        <ul class="menu">
+          <li class="nav-item"><a href="index.html">Home</a></li>
+          <li class="nav-item"><a href="#">About </a></li>
+          <li class="nav-item"><a href="#program">Program</a></li>
+          <li class="nav-item"><a href="#join">Join</a></li>
+          <li class="nav-item"><a href="#sponsor">Sponsor</a></li>
+          <li class="nav-item"><a href="#contact">News</a></li>
+          <li class="nav-item" id="campaign">
+            <a href="#campaign">ETA Campaign</a>
+          </li>
+        </ul>
+      </nav>
+      <!-- mobile menu-->
+      <div class="mobile-menu hidden">
+        <span class="close-menu"
+          ><img src="images/Iconmenu-close.png" alt="close menu icon"
+        /></span>
+        <div class="mobile-menu-content">
+          <a href="index.html">Home</a>
+          <a href="#">About</a>
+          <a href="#program">Program</a>
+          <a href="#join">Join</a>
+          <a href="#sponsor">Sponsor</a>
+          <a href="#news">News</a>
+          <a href="#campaign">ETA Campaign</a>
+        </div>
+      </div>
+    </header>
+    <main>
+      <!-- about hero section-->
+      <section class="about-hero">
+        <article class="hero-header">
+          <p class="greeting">"Hello! Web Developers"</p>
+          <h1 class="welcome">2023 Summer Web Developer conference</h1>
+        </article>
+        <p class="about-text">
+          The Global Summit brings together the community of experts, academics,
+          and activists who comprise the Creative Commons affiliate network in a
+          different country every two years. This year, we hope to expand our
+          invitation list including organizations and individuals.
+        </p>
```

Comment:
- [ ] It will be better to reduce the width of this paragraph by giving it `width:70%;` to be consistent with Figma design